### PR TITLE
[prometheus-rabbitmq-exporter] Updated rabbitmq exporter maintainer list

### PR DESCRIPTION
**Pull request by iamabhishek-dubey**
Signed-off-by: iamabhishek-dubey <abhishekbhardwaj510@gmail.com>

#### What this PR does / why we need it:

This PR is for adding @iamabhishek-dubey as maintainer for RabbitMQ exporter

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -24,7 +24,7 @@
 /charts/prometheus-pingdom-exporter/ @monotek @rpahli
 /charts/prometheus-postgres-exporter/ @gianrubio @zanhsieh
 /charts/prometheus-pushgateway/ @cstaud @gianrubio
-/charts/prometheus-rabbitmq-exporter/ @juanchimienti
+/charts/prometheus-rabbitmq-exporter/ @juanchimienti @iamabhishek-dubey
```
Comment (scottrigby) [inline .github/CODEOWNERS:27]: This should be sorted. There is a script in this repo you can run that should do this for you locally based on maintainers listed in the charts

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by scottrigby** [comment]
Relates to #21

**Comment by iamabhishek-dubey** [comment]
I have made the changes @scottrigby

**Review by torstenwalter — APPROVED** [review]

**Comment by torstenwalter** [comment]
Another maintainer of the chart needs to approve and can merge it directly afterwards.

**Comment by scottrigby** [comment]
Yep, per PROCESSES.md

**Comment by torstenwalter** [comment]
@scottrigby you still have a review, where you request changes. Can you dismiss that one or approve as well?

**Review by monotek — APPROVED** [review]

**Review by scottrigby — APPROVED** [review]

**Comment by scottrigby** [comment]
@Juanchimienti just checking in. Ideally a current maintainer of a chart would sign off on adding any additional maintainers. In our PROCESSES doc we say a repo administrator can decide to merge a PR if maintainers of the chart don't reply within 2 weeks. However adding a new maintainer should probably wait a bit longer (maybe we should specify that in PROCESSES). This PR has been open for a month.

In general I'd recommend only adding maintainers who have contributed non-insignificant PRs to the repo in the past. @iamabhishek-dubey has contributed to this repo a bit: one PR to add the Druid exporter (https://github.com/prometheus-community/helm-charts/pull/114). We also have a goal of having more than one maintainer for each chart (#21), and this chart has only one maintainer. Let's wait just a bit longer for @Juanchimienti to reply. But at a certain point, it would be good to have at least one other active maintainer for this chart.

**Comment by monotek** [comment]
@scottrigby seems we should just move forward with this one.

**Comment by scottrigby** [comment]
Agreed 👍

**Review by scottrigby — APPROVED** [review]

**Pull request merged**